src/components/Typography.css

```diff
@@ -0,0 +1,43 @@
+@import url('https://fonts.googleapis.com/css2?family=Livvic:ital,wght@0,600;0,700;1,600;1,700&display=swap');
+
+body {
+  /* font fallback stack from React defaults */
+  font-family: "Livvic", -apple-system, BlinkMacSystemFont, 'Segoe UI',
+    'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans',
+    'Helvetica Neue', sans-serif;
+  font-weight: 600; /* semi-bold */
+}
+
+h1, h2, h3 {
+  font-weight: 700; /* bold */
+}
+
+h1 {
+  font-size: 64px;
```

Comment:
The website needs to be responsive. Can we use units like rem here? So that it gets adjusted with the screen size

Reply:
Short answer: yes, fonts can adjust for different screen sizes.

Long answer: 
`rem` units aren't based on screen size but rather the font size of the root element. There are ways to automatically scale the size with things like `calc(10px + 2vmin)` but we don't actually want to have it constantly scaling with the size of the window. (`vmin` is based on the smallest dimension of the viewport.)

Figma shows different sizes at different breakpoints, and I think it makes sense to add media queries to adjust the font size of some elements to match the design at the tablet and mobile sizes.

Reply:
Breakpoints: 
- Desktop: Over 768px wide
- Tablet: 768px max width, ??? min width
- Mobile: ??? max width—Figma mobile designs are 375px wide, but that seems a little narrower than we may want to set the breakpoint?

Reply:
After discussion in Slack, we're going with: 

- Desktops (960px and above)
- Tablets (768px - 959px)
- Smartphones (below 768px)